Sources/Preferences/PreferencePane.swift

```diff
@@ -1,6 +1,6 @@
 import Cocoa
 
-public struct PreferencePaneIdentifier: Hashable, RawRepresentable, Codable {
+public struct PreferencePaneIdentifier: Hashable, RawRepresentable {
```

Comment:
?

Reply:
…huh. This might've been an accident that happened when I merged the master branch into my branch.